velox/connectors/hive/HiveDataSource.h

```diff
@@ -54,7 +54,7 @@ class HiveDataSource : public DataSource {
       const std::shared_ptr<common::Filter>& filter) override;
 
   uint64_t getCompletedBytes() override {
-    return ioStats_->rawBytesRead();
+    return ioStats_->read().sum();
```

Comment:
Direct input stream update rawBytesRead_ in DirectInputStream::loadSync()? Thanks!

Reply:
@xiaoxmeng Thanks for your review. Yes. The `rawBytesRead_ `is updated in `DirectInputStream::loadSync()`. But the `DirectInputStream::loadSync()` method is called with this [condition ](https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/DirectInputStream.cpp#L185). If it is not called, the `rawBytesRead_  `metric will be 0.